docs/README.md

```diff
@@ -0,0 +1,30 @@
+# Carbon documentation
+
+Welcome to the Carbon documentation. Here you'll find information on project
```

Comment:
```suggestion
This is the home for all Carbon documentation not related to tooling. These docs include information on project
```

Reply:
@connor-leech Why call out "not related to tooling"?

Reply:
Because this isn't an exhaustive repository of our documentation, it's primarily docs that don't live on our website.

Reply:
Do we have "tooling" documentation elsewhere?

Reply:
Our website? How to install Sketch, how to use our grid, icons, etc. Component usage guidance. 

If you think there's a better word than tooling I'd prefer it, as I typically don't like using that word.

Reply:
The /docs folder is for process, contribution, and version-related documentation. The website is usage guidelines and resources list.

Reply:
Brilliant, just updated 👍